ostree-simplified-installer.sh

```diff
@@ -527,6 +539,14 @@ groups = []
 
 [customizations]
 installation_device = "/dev/vda"
+
+[[customizations.user]]
```

Comment:
Why do you add this here?

Reply:
This is because the user defined in iot-container does not work, please see details in this bug https://github.com/osbuild/osbuild-composer/issues/3526 
(This bug was closed before due to the conversation with dev, they won't fix it for now. I think that's ok because user in commit/container is not recommended.)

So I need to add new user in iot-simplified-installer blueprint, and then use ${BLUEPRINT_USER} to execute upgrade/reboot commands.